lib/internal/errors.js

```diff
@@ -1483,6 +1483,15 @@ E('ERR_PACKAGE_PATH_NOT_EXPORTED', (pkgPath, subpath, base = undefined) => {
   return `Package subpath '${subpath}' is not defined by "exports" in ${
     pkgPath}package.json${base ? ` imported from ${base}` : ''}`;
 }, Error);
+E('ERR_PARSE_ARGS_INVALID_OPTION_VALUE', '%s', TypeError);
```

Comment:
Reading about errors, the description of `TypeError` is
> Indicates that a provided argument is not an allowable type. For example, passing a function to a parameter which expects a string would be a `TypeError`.

Probably all of the PARSE_ARGS errors should extend `Error` rather than `TypeError`?

Reply:
And just to be clear, this is an ECMAScript thing that you see in the browser too. It's not a Node.js oddity. As an example, `new URL('#')` returns a `TypeError` in the browser (and in Node.js).

Reply:
I wasn't thinking of the code being run as such. Ok, that now makes sense to me when the arguments are explicit, and by extension not unreasonable when implicit.